frontend/src/pages/LandingPage/Assets/img/Node Graph.svg

```diff
@@ -0,0 +1,34 @@
+<svg width="723" height="505" viewBox="0 0 723 505" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is this svg being used? Probs be better to call this over writing the graph svg inline to what you had right now

Reply:
y